src/mon/MgrStatMonitor.cc

```diff
@@ -69,6 +69,14 @@ void MgrStatMonitor::create_initial()
 void MgrStatMonitor::calc_pool_availability()
 {
   dout(20) << __func__ << dendl;
+
+  // if feature is disabled by user, do not update the uptime 
+  // and downtime, exit early
+  if (!g_conf().get_val<bool>("enable_availability_tracking")) {
```

Comment:
since that function is called every 5 seconds, the get_val will add a lot of processing. maybe consider use add_observer , that will make less calls